src/Symfony/Bridge/PhpUnit/ClockMock.php

```diff
@@ -71,7 +63,7 @@ public static function usleep($us)
         }
     }
 
-    public static function microtime($asFloat = false)
+    public static function microtime($asFloat = false): string
```

Comment:
this is a wrong return type

Reply:
I understand, but what type of float is recommended?

Reply:
this should use `@return string|float`, as done for the signature of `microtime`. But this must be done in phpdoc due to the union type.